### PR TITLE
Remove deprecated packages and packages that have been re-homed

### PR DESCRIPTION
**Pull request by benjdlambert**

### CODE REVIEW

File: docs/plugins/call-existing-api.md [hunk 1]
```diff
@@ -137,9 +137,8 @@ router.use('/summary', async (req, res) => {
 ```
 
 For a more detailed example, see
-[the lighthouse plugin](https://github.com/backstage/backstage/tree/master/plugins/lighthouse)
-that stores some state in a database and adds new capabilities to the underlying
-API.
+[the user-settings plugin backend](https://github.com/backstage/backstage/tree/blam/remove-deprecated-packages/plugins/user-settings-backend)
```
Comment (adamdmharvey) [inline docs/plugins/call-existing-api.md:140]: Hard coded to Ben's branch?

**Review by adamdmharvey — APPROVED** [review]
Looks great. TBH, surprised by the  movement of cost insights out. always felt that was an important feature, though its complexity to customize to cloud provider likely reduces value for brand new users, so I get it!

Minor find re: a user branch but LGTM!

**Comment by Rugvip** [comment]
@adamdmharvey our selection isn't really based on value, bit rather how central the plugin is to Backstage and whether there are other plugins building on top. For cost-insights it's a fairly standalone feature, so we opted for moving it over, hoping that makes sense?

**Review by Rugvip — APPROVED** [review]
:shipit:

**Review by vinzscam — APPROVED** [review]
💣

**Pull request merged**